src/activities/act-7-3-2.xml

```diff
@@ -136,9 +136,87 @@
   <answer>
     <p>
       <ol label="a.">
+        <li>
+          <figure xml:id="F-7-3-act-2-slope-field-answer">
+            <caption>The slope field of the given differential equation.</caption>
+            <image width="50%" source="images/7_3_Act2_slope_field_soln" />
+          </figure>
+
+        </li>
+
+        <li>
+          <p>
+            <m>y = 0</m> or <m>y = 6</m>.
+          </p>
+        </li>
+
         <li>
           <p>
-            Solutions for each of the prompts above.
+            The solution will tend to <m>y = 6</m>.
+          </p>
+        </li>
+
+        <li>
+          <sidebyside widths="47% 47%" margins="0%">
+            <table xml:id="T-7-3-act-2-s">
```

Comment:
The `@xml:id` here and on line 210 are already defined elsewhere. Since this is inside `answer`, I would err on the side of using something without captions, so no `table` or `figure` inside your `sidebyside` Just use the text before the `sidebyside` to say what's going on in the table at the left below and the image at the right below.